frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -138,6 +139,8 @@ export const valuesToApiFormat = (
     const parcelApiValues = landValuesToApiFormat({ data: p } as any);
     if (!!agencyId && p.agencyId === '') {
       parcelApiValues.agencyId = agencyId;
+    } else {
+      parcelApiValues.agencyId = +(parcelApiValues.agencyId as any).value;
```

Comment:
the agency Id is currently an object as set by the new agency component. Already discussed this with Johnny, one of us will take a longer look at fixing this time permitting.